nav2_dwb_controller/dwb_core/src/dwb_local_planner.cpp

```diff
@@ -253,9 +257,15 @@ DWBLocalPlanner::computeVelocityCommands(
     geometry_msgs::msg::TwistStamped cmd_vel;
     cmd_vel.twist = nav_2d_utils::twist2Dto3D(cmd_vel2d.velocity);
     return cmd_vel;
-  } catch (const nav2_core::PlannerException & e) {
+  } catch (const nav2_core::ControllerTFError & e) {
     pub_->publishEvaluation(results);
-    throw;
+    throw e;
+  } catch (const nav2_core::InvalidPath & e) {
+    pub_->publishEvaluation(results);
+    throw e;
+  } catch (const nav2_core::ControllerException & e) {
```

Comment:
Since they all derive from `ControllerException` and they are all handled the same, I think only this is required - though I actually don't know when its re-thrown if it gets rethrown by the base `ControllerException` type or the derived type

Reply:
I just tested, 

It strips the class back to `ControllerException` so I think we have to keep it for now :(.